src/main/java/bio/terra/service/snapshotbuilder/utils/ConceptChildrenQueryBuilder.java

```diff
@@ -68,7 +60,9 @@ public Query buildConceptChildrenQuery(
             descendantConceptId);
 
     // COUNT(DISTINCT person_id)
-    FieldVariable count = domainOccurrence.makeFieldVariable(PERSON_ID, "COUNT", "count", true);
+    FieldVariable count =
+        domainOccurrence.makeFieldVariable(
```

Comment:
I think all occurrence tables would have a `count()` method that does this

Reply:
Yes, this is in my next iteration